app/models/course/lesson_plan/item.rb

```diff
@@ -60,6 +63,18 @@ class Course::LessonPlan::Item < ApplicationRecord
     where(published: true)
   end)
 
+  scope :eager_load_personal_times_for, (lambda do |course_user|
```

Comment:
If there isn't going to be a `personal_times_for` scope that does not eager load, let's just name this `personal_times_for`?

Same for all the other `eager_load_*` functions.

Reply:
`eager_loaded_personal_times_for` will load the personal_times association if it has not loaded. But calling that method `personal_times_for` is leading because it will give wrong results if the association has been loaded for a __different__ course user. Hence I'm being explicit on both `eager_load_xx` and `eager_loaded_xx` methods.